internal/circuit/transport_test.go

```
@@ -0,0 +1,584 @@
+		Enabled:             true,
+		FailureThreshold:    3,
+		WindowSeconds:       60,
+		CooldownSeconds:     300,
+		MaxTransientRetries: 1, // small for tests
+		MaxRateLimitRetries: 1,
+	}.Defaults()
+	store := NewMemoryStore(cfg)
+	return NewTransport(inner, store, cfg, "openai", nil)
+}
+
+func newTestTransportWithStore(inner http.RoundTripper, store Store) *Transport {
```

golangci-lint (2.11.4)

[error] func newTestTransportWithStore is unused

(unused)